Desktop/components/JASP/Widgets/PlotEditor/PlotEditor.qml

```diff
@@ -81,61 +81,68 @@ Popup
 					contentHeight:			flickChild.height
 					contentWidth:			flickChild.width
 					flickableDirection:		Flickable.VerticalFlick
-					
-					
+										
 					onFlickStarted:			forceActiveFocus();
 					
 					Item
 					{
 						id:					flickChild
 						width:				axesFlickable.width
-						height:				yAxis.y + yAxis.height + jaspTheme.generalAnchorMargin
-					
-					
-						PlotEditingAxis
+						height:				xAxis.y + xAxis.height + jaspTheme.generalAnchorMargin
+
+
+						JASPC.DropDown
 						{
-							id:				xAxis
-							title:			qsTr("x-axis")
-							axisModel:		plotEditorModel.xAxis
-		
+							id:		axisDropDown
+							label: qsTr("Which axis should be shown?")
+							values:
+							[
+								{ label: qsTr("x-axis"),		value:	"x-axis"		},
+								{ label: qsTr("y-axis"),		value:	"y-axis"		}
+							]
+							startValue: "x-axis"
+
 							anchors
 							{
 								top:		parent.top
 								left:		parent.left
 								right:		parent.right
 								margins:	jaspTheme.generalAnchorMargin
-		
 							}
 						}
-		
-						Rectangle
+					
+						PlotEditingAxis
 						{
-							id:				axisSeparator
-							height:			1
-							color:			jaspTheme.uiBorder
+							id:				xAxis
+							title:			qsTr("x-axis")
+							axisModel:		plotEditorModel.xAxis
+							visible:		axisDropDown.value === "x-axis"
```

Comment:
It would be better to add an Enum for the axis and select it based on that.

Reply:
To make the selected axis better available here you could add a property to `PlotEditorModel` called `selectedAxis` and have that be a reference to the `PlotEditorAxisModel*` that is current.

But maybe it is better if I implement that.

Reply:
I changed it, let me know if you have any further comments!